src/scss/initial/_scale.scss

```diff
@@ -1,3 +1,4 @@
+@use 'sass:math';
 @use '../config/scale';
 @use '~accoutrement/sass/tools';
```

Comment:
@mirisuzanne @dvdherron I am curious to learn more about how we want to structure this moving forward. I know in TimeDesigner we used the `config` folder when we needed it (or tools) and forwarded & used `tools` in the config index. Are we not doing that here because this site was already set up with Sass modules and we are doing a quick update to Accoutrement? Or is there more that we consider when deciding how to structure that?

Reply:
I don't think we've discussed it in detail, so I think it's the 'quick update' that's happening here. I'd be happy to talk through what pattern we want to use, maybe set up a meeting to discuss?

Reply:
@stacyk @mirisuzanne I put a date on the calendar this Thursday. Maybe then we can touch on this and some of the remaining Accoutrement update/testing work?